core/core.go

```diff
@@ -367,6 +367,9 @@ func (n *IpfsNode) SetupOfflineRouting() error {
 	}
 
 	n.Routing = offroute.NewOfflineRouter(n.Repo.Datastore(), n.PrivateKey)
+
+	n.Namesys = namesys.NewNameSystem(n.Routing)
```

Comment:
maybe instead of putting it here, just move it out of `StartOnlineServices` and into `Standard`? not sure.